assembly/assembly-ide-war/pom.xml

```diff
@@ -287,6 +287,24 @@
             <artifactId>plugin-terminal-ui</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.eclipse.lsp4j</groupId>
+            <artifactId>org.eclipse.lsp4j</artifactId>
+            <exclusions>
```

Comment:
Because the "normal" library uses a ThreadLocal, which is not available in the gwt jre emulation. The xtext folks have made a special version of the runtime library to run under gwt.